libs/src/services/contracts/GovernedContractClient.js

```diff
@@ -41,18 +22,9 @@ class GovernedContractClient extends ContractClient {
    * Similar to `getMethod`
    */
   async getGovernedMethod (methodName, ...args) {
-    const web3 = this.web3Manager.getWeb3()
-
     const contractMethod = await this.getMethod(methodName, ...args)
-
-    const argumentTypes = contractMethod._method.inputs.map(i => i.type)
-    const argumentValues = contractMethod.arguments
-
-    const signature = createMethodSignature(methodName, argumentTypes)
-    const callData = abiEncode(web3, argumentTypes, argumentValues)
-
-    const contractRegistryKey = web3.utils.utf8ToHex(this.contractRegistryKey)
-
+    const { signature, callData } = this.governanceClient.getSignatureAndCallData(methodName, contractMethod)
```

Comment:
im not sure i understand how this has a reference to governanceClient? it looks like it `extends ContractClient` but i don't see us adding that to contractClient base

Reply:
it's in the ctor for GovernedContractClient (this class)

Reply:
oh gotcha this makes perfect sense actually - each governedContractClient has to expose some operations to the guardian execute tx through governance